### ISSUE TITLE
Looping an audio track causes audio buzz

### ISSUE THREAD

**Issue by spamatica** [muse-sequencer/muse#767]
Looping an audio track with current master causes the audio to seemingly get stuck on one buffer.

**Comment by terminator356** [comment]
This is with latency correction enabled, correct?
It's a known problem with the latency correction code I'm trying to fix.
Otherwise should not happen.

**Comment by spamatica** [comment]
That does not seem to be the issue, it seems to be off

Den tis 11 feb. 2020 kl 19:10 skrev Tim <notifications@github.com>:

**Comment by spamatica** [comment]
Retried this now and maybe there was something wrong with the state of latency correction. It was off in the menu. 
Trying now I can get the problem with latency correction but not without.
So as long as we don't recommend Latency correction this is not that big of an issue

**Comment by terminator356** [comment]
Ah I see. OK good (well not really).
Yeah I must get in there and return to the latency looping fixes.
It was complicated. I need to 'split' the audio pre-fetch into not just one but potentially
 two accesses because the processing code needs to have access to
 more than one buffer so that it can 'look ahead'.

**Comment by spamatica** [comment]
All in due time. 
Since it's only with Latency correction enabled I'm thinking we don't let it stop a 3.1 release. So I will try to go ahead and make the 3.1 release asap.

**Comment by terminator356** [comment]
Yes and in my mind what 'saves' me for now and buys me some time
 is that from past experience most users use looping with pure midi.
It's a bit unusual to use looping with waves...

**Comment by terminator356** [comment]
The issue would still be there since I've not done any work in there to fix it yet.
